packages/react-components/src/components/Popover/Popover.module.scss

```diff
@@ -5,6 +5,7 @@
   display: none;
   max-width: 336px;
   min-width: 168px;
+  z-index: $stacking-context-level-popover;
```

Comment:
You need to import it before usage `@import '../../utils/StackingContextLevel';`